webapp-django/crashstats/supersearch/models.py

```diff
@@ -43,26 +40,19 @@
 
 
 def get_api_allowlist(include_all_fields=False):
-    def get_from_es(include_all_fields):
-        cache_key = "api_supersearch_fields"
-        fields = cache.get(cache_key)
-        if fields is None:
-            all_fields = SUPERSEARCH_FIELDS
-            fields = []
-            for meta in all_fields.values():
-                if (
-                    meta["name"] not in fields
-                    and meta["is_returned"]
-                    and (include_all_fields or not meta["permissions_needed"])
-                ):
-                    fields.append(meta["name"])
-            fields = tuple(fields)
-
-            # Cache for 1 hour.
-            cache.set(cache_key, fields, 60 * 60)
-        return {"hits": fields}
-
-    return functools.partial(get_from_es, include_all_fields)
+    """Returns an API_ALLOWLIST value based on SUPERSEARCH_FIELDS"""
+
+    all_fields = SUPERSEARCH_FIELDS
+    fields = []
+    for meta in all_fields.values():
+        if (
+            meta["name"] not in fields
+            and meta["is_returned"]
+            and (include_all_fields or not meta["permissions_needed"])
+        ):
+            fields.append(meta["name"])
+
+    return {"hits": fields}
```

Comment:
We can make this return a real value rather than a callable since all this stuff is static between deploys now.